app/models/variant_override.rb

```diff
@@ -3,8 +3,6 @@ class VariantOverride < ActiveRecord::Base
 
   acts_as_taggable
 
-  attr_accessor :import_date
```

Comment:
This was overriding the ActiveRecord getter/setter, which is why querying didn't work for me. When defining new accessor methods, shouldn't they have a different name than the database field?

Reply:
Did we mean [attr_accessible](https://apidock.com/rails/ActiveRecord/Base/attr_accessible/class) instead, perhaps :thinking: ?

Reply:
Looks fine to delete :+1: